### ISSUE TITLE
[DOC REQUEST] Test examples

### ISSUE THREAD

**Issue by charlesdeb** [lucatume/wp-browser#668]
**Did you search the documentation first?**
Oh yes! 

**What do you think is missing from the documentation?**
Some examples of what Integration and EndToEnd tests might look like. In v3 you were getting close to having a few of those (under levels of testing I think). The new v4 could do with quite a few more examples and explanations. I have been using wpbrowser for a couple of years now and thought I understood it - but it appears that maybe I don't! Or maybe v4 has changed it in ways I didn't expect? So, a few more examples (especially in the getting started section) would help.

**Do you have any examples of what you'd like to see added to the documentation?**
I'd like to see an example of what the EndToEnd.suite.yml and Integration.suite.yml files should look like, along with a couple of sample tests for each suite that demonstrate the whole file (including `namespace`, `use`, and function argument and class definitions), rather than just a specific function (eg `havePostInDatabase()` or `assertQueries()`) which are alread reasonably well documented.

**Comment by lucatume** [comment]
There is a hefty [corpus of tests](https://github.com/lucatume/wp-browser/tree/master/tests) that should provide a number of examples.

Initializing a new project will scaffold an `EndToEnd` and `Integration` suite that should provide a starting example, is that not working?

**Comment by charlesdeb** [comment]
Yes, I did get a starting example. I've also just looked at the tests now. I would have normally expected them to be testing the nitty gritty of _your_ code, but I see you have quite a few tests there that show how I should use wpbrowser to test my code. Those would be very helpful to someone new to wpbrowser.

In that case, my recommendation would be to just put a reference to these tests at the bottom of your getting started page so that others can see them and get a good idea on how to use wpbrowser.
